src/app/components/LndHub/LndHub.stories.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import LndHub from ".";
```

Comment:
what's actually a good convention here: As this is a form should this be called `LndHubForm`

Reply:
That would be a good convention. And maybe bundle it in a Forms folder for e.g. if there are many.